src/db/tables/onerep_scans.ts

```diff
@@ -69,47 +92,9 @@ async function setOnerepManualScan(
 
 async function addOnerepScanResults(
   onerepProfileId: number,
-  onerepScanId: number,
   onerepScanResults: Array<ScanResult>,
-  onerepScanReason: Scan["reason"],
-  onerepScanStatus: Scan["status"],
 ) {
   await knex.transaction(async (transaction) => {
```

Comment:
I think this no longer needs to be in a transaction, since it's just a single query?

Reply:
Good point, it shouldn't be needed now.